qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -217,7 +217,11 @@ def generate_unroll_3q(
             target=target,
         )
     )
-    unroll_3q.append(HighLevelSynthesis(hls_config=hls_config))
+    unroll_3q.append(
+        HighLevelSynthesis(
+            hls_config=hls_config, coupling_map=None, target=target, use_qubit_indices=False
```

Comment:
This should be safe to do now that we have `use_qubit_indices=False`. This enables you to do something like find the longest path for LNN synthesis before layout/routing.

```suggestion
            hls_config=hls_config, coupling_map=coupling_map, target=target, use_qubit_indices=False
```